packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -379,6 +380,10 @@ export default class Meetings extends WebexPlugin {
         StaticConfig.set(this.config);
         LoggerConfig.set(this.config.logging);
         LoggerProxy.set(this.webex.logger);
+
+        // SDK need to perform reachability check on app load
+        this.startReachability();
```

Comment:
Should we wait to trigger the meetings ready until reachability is completed? @arun3528?

Reply:
logically it should finish before addMEdia , I am assuming the join meetingInfo should take more then 5 -6 min which is sufficient for the getting reachability data